test/fixtures/sidecar-containers/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
           successThreshold: 1
           timeoutSeconds: 5
         name: hello-world
+        args: [something]
```

Comment:
Let's add `env` also?

Should we have an explicit test to validate that this key is not being sent?

Reply:
the unit test of `metadata-extractor.spec.ts` should validate that?

Reply:
I see what you mean now, sorry. I will add a test now 😅